From 32f380a1128371265a3fa5dd2fdb318bfe8630e9 Mon Sep 17 00:00:00 2001
From: Claude Heiland-Allen <claude@mathr.co.uk>
Date: Fri, 9 Jun 2023 15:48:39 +0100
Subject: [PATCH] ensure the class dispatched files are included in dist
 tarballs

---
 src/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index da62435e..80dd7b8d 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -204,6 +204,7 @@ pkginclude_HEADERS = m_pd.h m_imp.h g_canvas.h g_undo.h g_all_guis.h s_stuff.h \
 include_HEADERS = m_pd.h
 noinst_HEADERS = s_audio_alsa.h s_audio_paring.h s_utf8.h m_private_utils.h
 noinst_HEADERS += z_hooks.h z_ringbuffer.h x_libpdreceive.h
+noinst_HEADERS += m_class_dispatcher_1.h m_class_dispatcher_2.h
 
 if LIBPD
 libpdinclude_HEADERS += m_pd.h z_libpd.h
@@ -215,7 +216,8 @@ endif
 # we want these in the dist tarball
 EXTRA_DIST = CHANGELOG.txt notes.txt pd.rc \
     makefile.gnu makefile.mac makefile.mingw makefile.msvc \
-    d_soundfile.h s_audio_audiounit.c s_audio_esd.c
+    d_soundfile.h s_audio_audiounit.c s_audio_esd.c \
+    m_class_dispatcher.c
 
 # add WISH define if it's set
 WISH=@WISH@
-- 
2.44.0

